pipelines/tekton/gitops-update-pipeline/gitea-edge-user-1-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Do these `gitea-edge-user-*` secrets need to be checked in to the repo OR should we document them as a required resource for the PipelineRun?

Reply:
That's a good question, and the same goes for the PVCs for the PipelineRun (don't actually need a dedicated PVC even...tekton can generate one per PipelineRun). We currently don't deploy the PipelineRuns by default, but we do provision these Secrets and PVCs. I think maybe we should check them in as examples that are easy to apply for our purposes (and easy to change for others who want to run the demo on their own infra or base something else on it), and document that there are examples that people can change (and that they should consider not checking-in those changes).

WDYT?

Reply:
Removed automatic deployment of these secrets, and made it clearer that they're examples.